diesel_compile_tests/tests/compile-fail/associations_errors.rs

```diff
@@ -0,0 +1,32 @@
+#[macro_use] extern crate diesel;
+
+#[derive(Associations)]
+//~^ ERROR Could not derive `Associations` without any `#[belongs_to(table_name)]` attribute
+struct Post1 {
+    a: i32,
+}
+
+#[derive(Associations)]
+//~^ ERROR The foreign key column post_id is not found
```

Comment:
Can we improve this error message to say "No field found that corresponds to the column `post_id`"